src/cdfmean.f90

```diff
@@ -115,10 +124,15 @@ PROGRAM cdfmean
      PRINT *,'                   partial steps.'
      PRINT *,'       [ -var ]  : also compute the spatial variance of cdfvar '
      PRINT *,'       [ -zeromean ] : create a file with cdfvar having a zero spatial mean.'
+     PRINT *,'       [ -out_suff OUT-file-suffix ] : specify SUFFIX (+ directory) of the output files'
```

Comment:
I didn't really catch it before reading the code. I suggest something like "specify directory + prefix to the output name (OUT-file-suffix+cfout)"